src/odemis/gui/cont/streams.py

```diff
@@ -2534,6 +2533,36 @@ def _updateMicroscopeStates(self):
                 self._tab_data_model.emState.value = guimodel.STATE_OFF
 
 
+class CryoStreamsController(StreamBarController):
+    """
+    Controls the display of stream panels without affecting the actual streams
+    """
+
+    def _onView(self, view):
+        """ Handle the changing of the focused view """
+        if not view or self.ignore_view:
+            return
+
+        # hide/show the stream panels which are compatible with the view
+        allowed_classes = view.stream_classes
+        for e in self._stream_bar.stream_panels:
+            ov_streams = self._tab_data_model.overviewStreams
+
+            if view.__class__ == guimodel.FeatureView and ov_streams.value:
```

Comment:
You should typically never use `__class__`. Most often you wan to use `isinstance(view, ...)`, but may you actually don't want to catch the subclasses. In this case you can write `type(view) == ...`.